src/prep_helpers.py

```
@@ -160,25 +160,71 @@ async def detect_disc_and_category(prep_instance: Any, meta: Meta) -> tuple[str,
+                        choice = cli_ui.ask_choice(
+                            "Choose category for audio release:",
+                            choices=["1. Music", "2. Audiobook"],
+                        )
+                        if choice.startswith("1") or choice.lower() == "music":
+                            meta.category = "MUSIC"
+                            meta.audiobook = False
+                        else:
+                            meta.category = "BOOK"
+                            meta.audiobook = True
+                        logger.info(f"[cyan]Category selected interactively: {meta.category}[/cyan]")
+                    except Exception:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)